src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/kjerne/vilkårsvurdering/domene/VilkårsvurderingRepositoryTest.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.ks.sak.no.nav.familie.ks.sak.kjerne.vilkårsvurdering.domene
+
+import no.nav.familie.ks.sak.OppslagSpringRunnerTest
+import no.nav.familie.ks.sak.data.lagVilkårsvurdering
+import no.nav.familie.ks.sak.data.randomAktør
+import no.nav.familie.ks.sak.kjerne.vilkårsvurdering.domene.Resultat
+import no.nav.familie.ks.sak.kjerne.vilkårsvurdering.domene.VilkårsvurderingRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+
+internal class VilkårsvurderingRepositoryTest : OppslagSpringRunnerTest() {
+
+    @Autowired
+    private lateinit var vilkårsvurderingRepository: VilkårsvurderingRepository
+
+    @BeforeEach
+    fun beforeEach() {
+        opprettSøkerFagsakOgBehandling(randomAktør())
+    }
+
+    @Test
+    fun `finnAktiv - skal returnere aktiv vilkårsvurdering for behandling`() {
+        val vilkårsvurdering = lagVilkårsvurdering(søker, behandling, Resultat.IKKE_VURDERT)
+        vilkårsvurderingRepository.saveAndFlush(vilkårsvurdering)
+
+        val hentetVilkårsvurdering = vilkårsvurderingRepository.finnAktiv(behandling.id)
+
+        assertNotNull(hentetVilkårsvurdering)
+        assertEquals(behandling.id, hentetVilkårsvurdering!!.behandling.id)
```

Comment:
Vi har en extension metode `.shouldNotBeNull()` som man kunne ha brukt her.
Da slipper man eksplisitt !! sjekk senere :D 

```suggestion
        val hentetVilkårsvurdering = vilkårsvurderingRepository.finnAktiv(behandling.id).shouldNotBeNull()

        assertEquals(behandling.id, hentetVilkårsvurdering.behandling.id)
```

Reply:
Aha! 👍